pkg/kapp/app/recorded_app.go

```diff
@@ -77,6 +77,51 @@ func (a *RecordedApp) UpdateUsedGVs(gvs []schema.GroupVersion) error {
 	})
 }
 
+func (a *RecordedApp) UsedGVKs() ([]schema.GroupVersionKind, error) {
+	meta, err := a.meta()
+	if err != nil {
+		return nil, err
+	}
+
+	return meta.UsedGVKs, nil
+}
+
+func (a *RecordedApp) UpdateUsedGVKs(gvks []schema.GroupVersionKind, merge bool) ([]schema.GroupVersionKind, error) {
+	gvksByGVK := map[schema.GroupVersionKind]struct{}{}
```

Comment:
nit: We can call it uniqGVKsMap

Reply:
I do not see why not, stuck to this because of [this](https://github.com/vmware-tanzu/carvel-kapp/blob/develop/pkg/kapp/app/recorded_app.go#L65) existing precedent